__tests__/app.test.js

```diff
@@ -200,6 +200,24 @@ describe("7. POST /api/reviews/:review_id/comments", () => {
         expect(body.msg).toBe("id not found");
       });
   });
+  test("key username is spelt incorrectly", () => {
```

Comment:
additional test for the object within .send()  where the object has an additional unwanted key, without side effects.